lib/buffer.js

```diff
@@ -157,6 +157,12 @@ function palloc(that, length) {
   var buf = sliceOnto(allocPool, that, start, end);
   poolOffset = end;
 
+  // Ensure aligned slices
+  if (poolOffset & 0x7) {
+    poolOffset |= 0x7;
```

Comment:
1. `poolOffset = 1`
2. `poolOffset & 0x7 === 1`
3. `poolOffset |= 0x7` => `0x7`
4. `poolOffset++` => `0x8`
5. Voila, `poolOffset` is aligned!